### PR TITLE
Align on use of listener

### PR DESCRIPTION
**Pull request by rebloor**
### Description

Updates event description to use the term listener consistently when describing argument.

### Motivation

To avoid confusion, due to use of both listener and callback.

### Additional details

Also took the opportunity to shift language to present tense and generally simplify.

### CODE REVIEW

**Comment by rebloor** [comment]
@Rob--W @willdurand as you will see, this turned out to involve quite a number of files. I'll also check everything again myself to make sure I have not inadvertently added any new issues.

File: files/en-us/mozilla/add-ons/webextensions/api/commands/onchanged/index.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Events have three functions:
 
 - `listener`
 
-  - : Function that is called when a command's shortcut changes. The function is passed these:
+  - : The function called when a command's shortcut changes. The function is passed these attributes:
```
Comment (Rob--W) [inline files/en-us/mozilla/add-ons/webextensions/api/commands/onchanged/index.md:37]:
```suggestion
  - : The function called when a command's shortcut changes. The function is passed an object with these properties:
```

... or actually we should call the parameter `changeInfo`.

File: files/en-us/mozilla/add-ons/webextensions/api/devtools/panels/extensionsidebarpane/onshown/index.md [hunk 2]
```diff
@@ -30,8 +30,8 @@ Events have three functions:
 
 ### Parameters
 
-- `callback`
-  - : Function that will be called when this event occurs. The function will be passed no arguments.
+- `listener`
+  - : The function called when this event occurs. The function is passed no arguments.
```
Comment (Rob--W) [inline files/en-us/mozilla/add-ons/webextensions/api/devtools/panels/extensionsidebarpane/onshown/index.md:34]: (pre-existing) issue: documentation is wrong. It receives a parameter: https://searchfox.org/mozilla-central/rev/a4fd6daad3a4123d995478467c1274653b283801/browser/components/extensions/schemas/devtools_panels.json#245-258

File: files/en-us/mozilla/add-ons/webextensions/api/menus/create/index.md [hunk 3]
```diff
@@ -106,7 +106,7 @@ browser.menus.create(
     - `visible` {{optional_inline}}
       - : `boolean`. Whether the item is shown in the menu. Defaults to `true`.
 
-- `callback` {{optional_inline}}
+- `listener` {{optional_inline}}
```
Comment (Rob--W) [inline files/en-us/mozilla/add-ons/webextensions/api/menus/create/index.md:109]:
```suggestion
- `callback` {{optional_inline}}
```

This is a callback function, not a parameter to addListener/removeListener. Please double-check that you haven't inadvertently renamed other non-event parameters in this PR.

**Review by Rob--W — CHANGES_REQUESTED** [review]

File: files/en-us/mozilla/add-ons/webextensions/api/downloads/oncreated/index.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ Events have three functions:
 
 - `function`
 
-  - : A callback function that will be called when this event occurs. This function will be passed the following arguments:
+  - : The function called when this event occurs. This function will be passed this argument:
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/downloads/oncreated/index.md:37]:
```suggestion
  - : The function called when this event occurs. This function is passed this argument:
```

File: files/en-us/mozilla/add-ons/webextensions/api/downloads/onerased/index.md [hunk 5]
```diff
@@ -32,9 +32,9 @@ Events have three functions:
 
 ### Parameters
 
-- `callback`
+- `listener`
 
-  - : A callback function that will be called when this event occurs. This function will be passed the following arguments:
+  - : The function called when this event occurs. This function will be passed this argument:
```
Comment (rebloor) [inline files/en-us/mozilla/add-ons/webextensions/api/downloads/onerased/index.md:37]:
```suggestion
  - : The function called when this event occurs. This function is passed this argument:
```

**Review by Rob--W — APPROVED** [review]
I see that the issue I called out at https://github.com/mdn/content/pull/26953#discussion_r1204554194 has been addressed in https://github.com/mdn/content/pull/26967.

Thanks for the improvements!

**Pull request merged**